playbooks/support/responding.md

```diff
@@ -75,28 +94,93 @@ One of the on-call members handles internal and external communication for the i
-  have to be detailed (Not every stage of the incident necessarily need to be communicated on status page). Remember that this page is targeted to a public audience(i.e., users and partners) of Artsy products. Some might be subscribed to receive updates on every communication here. Use your judgement based on severity and duration of the incident. The most important thing is acknowledging the issue, specifying the services affected and communicating key updates and resolutions. Use the status page integration on the Jira Ops ticket for updating Status page.
-
-  _Jira integrations are known to be flakey. If you have trouble getting the integration to work, simply visit [Statuspage](https://statuspage.io), create an incident manually, and link to it from the ticket._
+- Resolution happens exclusively in the dedicated slack channel associated with the OpsGenie incident.
+- Meaningful updates, discoveries, or milestones (e.g. changes in availability or resolution) are added to the
+  OpsGenie incident by its responders.
+- Communicate progress and major milestones publicly through statuspage.io. Updates don't need to be specific, but
+  it is important that we give customers confidence that we are dealing with the incident.
 
 ## Step 5: Resolve
 
 Incidents are resolved when the immediate issue has been mitigated. After that, the team needs to make sure all
-follow-up items are tracked and the Jira Ops ticket clearly reflects this state.
+follow-up items are tracked and the OpsGenie incident clearly reflects this state.
 
 1. Create any necessary follow-up tickets either in the bug backlog or a specific team’s project board. Link these
-   from within the Jira Ops ticket.
-2. Post any final updates to the Jira Ops ticket
+   from within the OpsGenie incident.
+2. Post any final updates to the OpsGenie incident
 
-- Update timestamps
-- Mark the ticket as **RESOLVED**
-- Add any additional labels or metadata that may be relevant later
+   - Add a timeline entry describing why you're resolving the incident
+   - Mark the incident as **RESOLVED**
 
-3. Resolve any outstanding incidents on our status page via the integration on the Jira Ops ticket.
-4. If the incident was Critical, create a post mortem following the guidelines in our
-   [post_mortems repo](https://github.com/artsy/post_mortems).
+3. Resolve any outstanding incidents on our status page via statuspage.io.
+4. If the incident was Critical, create a post mortem in OpsGenie.
 5. Update relevant [playbooks](https://github.com/artsy/potential/wiki) 🔒 with any lessons.
+
+## Severity of Incidents
+
+### P1 - **Critical**
+
+Critical incidents have a very high impact:
+
+- A user-facing service is down for all users.
+- Confidentiality or privacy is breached.
+- Customer data loss.
+- “All hands on deck.”
+
+The **Critical** level in OpsGenie equates to the **Critical** impact level in statuspage.io.
```

Comment:
This one lines up pretty well, but the rest of the levels I guessed at. I decided on...

OpsGenie : StatusPage
Critical : Critical
High : Major
Moderate : Minor
Low : (none, doesn't warrant a public-facing incident)
Informational : (none, doesn't warrant a public-facing incident)

Let me know if you think I got these wrong!

Reply:
We might want to leave the mappings off for now. During the creation of an incident on StatusPage.io, you can't actually set the severity (which is only exposed to viewers of the page via the incident headline text color).

Instead, you set the status of components in your system and a severity is calculated for the incident based on the component statuses.

After creating the incident, you may override the calculated severity.

I think we owe some documentation on setting the appropriate status for components but we might want to stick with the calculated severity on StatusPage.io for now. What do you think?


source: https://help.statuspage.io/help/component-status-incident-impact-and-top-level-status-calculations

Reply:
Sure, that works. I'll remove those lines (and any references to them).